src/main/java/com/dungeoncrawler/Javiarenka/character/Character.java

```diff
@@ -41,4 +45,12 @@ public boolean isAlive() {
     public void setAlive(boolean alive) {
         isAlive = alive;
     }
+
+    public CharacterStatus getCurrentStatus() {
+        return currentStatus;
+    }
+
+    public void setCurrentStatus(CharacterStatus currentStatus) {
+        this.currentStatus = currentStatus;
+    }
```

Comment:
getter i setter są w porządku, ale jeśli zmienimy obiekt status na listę obiektów, to przydała by się metoda checkForStatus(String statusName)

Reply:
da się zrobić ;)